ckan/ckanext/ckanext-ytp_main/ckanext/ytp/views.py

```diff
@@ -118,6 +124,184 @@ def recent_datasets_feed():
+    organizations = total_results.get('organizations', {})
+    item_count = datasets.get('count', 0) + organizations.get('count', 0)
+
+    # get the amount of results for each type of resource and resources in separate queries
+    only_datasets = get_action('package_search')(context, {'q': q, 'fq': 'dataset_type:dataset'})
+    dataset_count = only_datasets.get('count', 0)
+    only_apisets = get_action('package_search')(context, {'q': q, 'fq': 'dataset_type:apiset'})
+    apiset_count = only_apisets.get('count', 0)
+    only_showcases = get_action('package_search')(context, {'q': q, 'fq': 'dataset_type:showcase'})
+    showcase_count = only_showcases.get('count', 0)
+    only_organizations = get_action('organization_search')(context, {'q': q})
+    organization_count = only_organizations.get('count', 0)
+
+    # combine the results
+    result_count['dataset'] = dataset_count
+    result_count['apiset'] = apiset_count
+    result_count['showcase'] = showcase_count
+    result_count['organization'] = organization_count
+    result_count['all'] = dataset_count + apiset_count + showcase_count + organization_count
+
+    dataset_sets = datasets.get('results', [])
+    organization_sets = organizations.get('results', [])
+
+    # get additional info for the organizations
+    for organization in organization_sets:
+        # Parse the organization description from the extras 
+        org_extras = organization.get('extras', [])
+        t = next(filter(lambda t: t.get('key') == 'description_translated', org_extras), None)
+        if t:
+            translations = t.get('value')
+            new_description_translated = json.loads(translations)
+            # add the translated description field
+            organization['description_translated'] = new_description_translated
+
+    # combine organizations and other resources
+    combined_results = organization_sets + dataset_sets
+
+    # determine the page amount
+    if dataset_type == 'all':
+        # count all but organizations
+        datasets = int(result_count.get('all', 0)) - int(result_count.get('organization', 0))
+    else:
+        datasets = int(result_count.get(dataset_type, 0))
+
+    # Use the higher count for the number of total pages
+    dataset_pages = int(math.ceil(datasets / dataset_limit))
+    organization_pages = int(math.ceil(float(result_count.get('organization', 0)) / organization_limit))
+    total_pages = dataset_pages if dataset_pages > organization_pages else organization_pages
+
+    g.general_search = {
+            u'q': q,
+            u'total_results': combined_results,
+            u'result_count': result_count,
+            u'item_count': item_count,
+            u"last_query": params_to_dict(request.form),
+            u'filter': chosen_filter,
+            u'page': page,
+            u'sort_string': sort_string,
+            # u'total_pages': int(math.ceil(float(result_count.get(dataset_type, 0)) / float(dataset_limit + organization_limit))),
```

Comment:
Probably good idea to describe why it's commented (or remove, if there isn't planned usage for this piece of code)

Reply:
Resolved in [92a479a](https://github.com/vrk-kpa/opendata/pull/1774/commits/92a479a6671ee9d1f0071c475d1135bf7a17fbd2)